tests/Place/Canonical/DBALDuplicatePlaceRepositoryTest.php

```diff
@@ -178,4 +193,27 @@ public function it_can_get_the_duplicates_of_a_place(): void
             $this->duplicatePlaceRepository->getDuplicatesOfPlace('64901efc-6bd7-4e9d-8916-fcdeb5b1c8ad')
         );
     }
+
+    /**
+     * @test
+     */
+    public function it_can_mark_a_place_as_processed(): void
+    {
+        $placeId = '19ce6565-76be-425d-94d6-894f84dd2947';
+
+        $this->duplicatePlaceRepository->markAsProcessed($placeId);
+
+        $actualRows = $this->connection->createQueryBuilder()
```

Comment:
This test is not so safe, it does not verify if not all places are marked as processed.